client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -117,7 +117,7 @@ export class DisplayTable extends React.Component {
       const current_col_formatter = col_configs_with_defaults[col].formatter;
       const current_col_raw_formatter =
         col_configs_with_defaults[col].raw_formatter;
-      if (current_col_formatter) {
+      if (current_col_formatter && !_.isFunction(current_col_formatter)) {
```

Comment:
Just wondering why did you need this one?

Reply:
Since I changed formatter to a function, it would float left since `_.isString(current_col_formatter)` returned false.

Reply:
This seems poorly written originally. Try this:
```     
return (
        (_.isString(current_col_formatter) && _.isNumber(row[col])) ||
        _.isNumber(current_col_raw_formatter(row[col]))
  ? "right" : "left"
)
```